lambda/authorizer/lambda_functions.py

```diff
@@ -18,13 +18,17 @@
 import ssl
 from typing import Any, Dict
 
+import boto3
 import create_env_variables  # noqa: F401
 import jwt
 import requests
 from utilities.common_functions import authorization_wrapper, get_id_token
 
 logger = logging.getLogger(__name__)
 
+secrets_manager = boto3.client("secretsmanager", region_name=os.environ["AWS_REGION"])
```

Comment:
let's add the retry config in here too- we have a method for that, we just need to import it. examples:
https://github.com/awslabs/LISA/blob/develop/lambda/models/lambda_functions.py#L24
https://github.com/awslabs/LISA/blob/develop/lambda/models/lambda_functions.py#L64-L67